### ISSUE TITLE
B-Tree Walker is broken

### ISSUE THREAD

**Issue by sdimitro** [delphix/sdb#297]
## Message

```
  File "/usr/lib/python3/dist-packages/sdb/command.py", line 651, in _call
    yield from self.walk(obj)
  File "/usr/lib/python3/dist-packages/sdb/commands/zfs/btree.py", line 86, in walk
    yield from self._helper(obj.bt_root)
  File "/usr/lib/python3/dist-packages/sdb/commands/zfs/btree.py", line 70, in _helper
    if node.bth_core:
AttributeError: 'zfs_btree_hdr_t' has no member 'bth_core'
----------------------------------------------------------
Link: https://github.com/delphix/sdb/issues/new
```

## Reason

The ZFS codebase changed in this commit: https://github.com/delphix/zfs/commit/c0bf952c846100750f526c2a32ebec17694a201b

## Fix

```
 69         count = node.bth_count
 -70         if node.bth_core:
 +70         if int(node.bth_first) == -1:
 71             # alterate recursive descent on the children and generating core objects
 72             core = drgn.cast('struct zfs_btree_core *', node)
 73             for i in range(count):
```

**Comment by sdimitro** [comment]
Repro: `sdb> spa | vdev 0 | metaslab | member ms_allocatable.rt_root | filter "obj.bt_num_elems != 0" | head 1 | zfs_btree`

**Comment by sdimitro** [comment]
Fixed in 6.0/stage. Should propagate to master soon
